packages/test-utils/src/utils.ts

```diff
@@ -270,3 +273,39 @@ export const toReadableStream = (...args: unknown[]): Readable => {
     })
     return rs
 }
+
+/**
+ * Used to spin up an HTTP server used by integration tests to fetch private keys having non-zero ERC-20 token
+ * balances in streamr-docker-dev environment.
+ */
+export class KeyServer {
+    public static readonly KEY_SERVER_PORT = 45454
+    private readonly server: http.Server
+
+    constructor() {
+        const app = express()
+        app.use(cors())
+
+        // try avoid sequential tests starting from same address
+        let c = (Math.round((Math.random() * 1000)) % 1000)
+        app.get('/key', (req, res) => {
+            c = ((c + 1) % 1000)
```

Comment:
this will also use 0 as a key. but that privatekey is invalid, since to get to the address at one point you need to devide through the key, and then it would devide through 0.
So 0 needs to be avoided.

Reply:
Right, I copied the client keyserver implementation here, looks like guess the broker version was slightly different but probably correct:

```ts
 const app = express()
    global.__StreamrKeyserver = new KeyServer()
    let c = 0
    app.get('/key', (_req, res) => {
        c += 1
        const hexString = c.toString(16)
        const privkey = '0x' + hexString.padStart(64, '0')
        log('key endpoint called, returning key ' + privkey)
        if (c > 1000 ) { c = 0 }
        res.send(privkey)
    })
    app.listen(45454)
```

Reply:
I'll update to use the above logic instead

Reply:
Good catch btw! :+1:

Reply:
@samt1803 applied your changes